### PR TITLE
[MIG] 10.0 port project_task_default_stage

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: project_task_default_stage/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {
     'name': 'Project Task Default Stage',
     'summary': 'Recovery default task stage projects from v8',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Project',
     'author': 'Incaser Informatica S.L., '
```
Comment (pedrobaeza) [inline project_task_default_stage/__manifest__.py]: Please switch the author to the new company `Tecnativa` instead of `Incaser...`.

**Review by pedrobaeza — APPROVED** [review]
Little comment, but is OK (code review)

**Comment by pedrobaeza** [comment]
Thanks

File: project_task_default_stage/models/__init__.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # -*- coding: utf-8 -*-
+from . import project_task_type
 from . import project
```
Comment (dreispt) [inline project_task_default_stage/models/__init__.py]: It would be best for the Python file to be named after the Model it implements, as originally.

**Review by dreispt — APPROVED** [review]

**Pull request merged**